### ISSUE TITLE
How to detect a single image

### ISSUE THREAD

**Issue by KevinHuang1102** [THU-MIG/RepViT#59]
Hello, I successfully trained my dataset
But I want to use the trained model to classify a single image
Want to ask how to do it
I tried changing --data path but can't seem to point to a single image

**Comment by seany56** [comment]
> Hello, I successfully trained my dataset But I want to use the trained model to classify a single image Want to ask how to do it I tried changing --data path but can't seem to point to a single image

Hello, I have recently encountered the same problem as you. Can you provide an email address to facilitate communication? To protect privacy, I will delete this request as soon as I receive the email address.

**Comment by wydxry** [comment]
because it has BN_Linear, it not works when batch_size =1,  you can change the BatchNorm1d to the LayerNorm
